R-package/configure.ac

```diff
@@ -28,8 +28,15 @@ fi
 
 if test `uname -s` = "Darwin"
 then
-  OPENMP_CXXFLAGS='-Xclang -fopenmp'
-  OPENMP_LIB='-lomp'
+  if command -v brew &> /dev/null
+  then
+    HOMEBREW_LIBOMP_PREFIX=`brew --prefix libomp`
+  else
+    # Homebrew not found
+    HOMEBREW_LIBOMP_PREFIX=''
+  fi
+  OPENMP_CXXFLAGS="-Xpreprocessor -fopenmp -I${HOMEBREW_LIBOMP_PREFIX}/include"
```

Comment:
what's the difference between -Xclang and -Xpreprocessor ?

Reply:
No idea. I only know that Xprocessor works on my Macbook but Xclang doesn't.

Reply:
I think it has to do with the fact that we have to specify extra flags to the preprocessor specifically, due to non-standard location of OpenMP.